packages/block-library/src/image/edit.js

```diff
@@ -481,15 +481,15 @@ class ImageEdit extends Component {
 									type="number"
 									className="block-library-image__dimensions__width"
 									label={ __( 'Width' ) }
-									value={ width !== undefined ? width : imageWidth }
+									value={ width ? width : imageWidth }
```

Comment:
Hi @kienstra thank you for submitting this PR to address this issue. imageWidth/imageHeight also don't contain a default value:
```
https://github.com/WordPress/gutenberg//blob/079c11b0d011c228718388e829c9826e1f34183a/packages/block-library/src/image/edit.js#L576-L579
```

Is it possible that because of imageWidth/imageHeight also being undefined we still get an error? Should we also add a default value of 0 to this variable?

Reply:
Hi @jorgefilipecosta,
Thanks, https://github.com/WordPress/gutenberg/pull/15239/commits/2655161018fcf6c1825b2dc0b8538046186cc2bd applies your suggestion. 

4 of the 5 times I tested it, the [console warning](https://github.com/WordPress/gutenberg/pull/15239#issue-274215441) didn't appear. When it did appear, maybe I forgot to do a hard refresh.